### PR TITLE
DOCS-643: Sources of system metrics.

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation

Detailed description:

- Restructured introduction to the System Tables topic.
- Added information about system metrics providers from #10544

### CODE REVIEW

File: docs/en/operations/system-tables.md [hunk 1]
```diff
@@ -5,11 +5,40 @@ toc_title: System Tables
...
-You can’t delete a system table (but you can perform DETACH).
-System tables don’t have files with data on the disk or files with metadata. The server creates all the system tables when it starts.
-System tables are read-only.
-They are located in the ‘system’ database.
+## Introduction
+
+System tables provide:
+
+- Information about server states, processes and environment.
+- Support for server's internal processes.
```
Comment (blinkov) [inline docs/en/operations/system-tables.md:13]:
```suggestion
System tables provide information about:

- Server states, processes, and environment.
- Server's internal processes.
```

File: docs/en/operations/system-tables.md [hunk 2]
```diff
@@ -5,11 +5,40 @@ toc_title: System Tables
...
+
+System tables:
+
+- Don't store data and metadata in the storage filesystem. ClickHouse server creates system tables at start.
```
Comment (blinkov) [inline docs/en/operations/system-tables.md:17]: Afaik not always true, some tables are persisted.

File: docs/en/operations/system-tables.md [hunk 3]
```diff
@@ -5,11 +5,40 @@ toc_title: System Tables
...
+- Located in the `system` database.
+- Available only for reading data.
+- Can't be deleted, but can be detached.
```
Comment (blinkov) [inline docs/en/operations/system-tables.md:20]: * Alter is also worth mentioning.
* Deleted -> dropped.

File: docs/en/operations/system-tables.md [hunk 4]
```diff
@@ -5,11 +5,40 @@ toc_title: System Tables
...
+
+### Sources of system metrics
```
Comment (blinkov) [inline docs/en/operations/system-tables.md:22]:
```suggestion
### Sources of System Metrics
```

File: docs/en/operations/system-tables.md [hunk 5]
```diff
@@ -5,11 +5,40 @@ toc_title: System Tables
...
+
+For collecting system metrics ClickHouse server uses:
+
+- `CAP_*_ADMIN` capabilities.
```
Comment (blinkov) [inline docs/en/operations/system-tables.md:26]: need some explanation/reference/wiki links

File: docs/en/operations/system-tables.md [hunk 6]
```diff
@@ -5,11 +5,40 @@ toc_title: System Tables
...
+- procfs (only in Linux).
```
Comment (blinkov) [inline docs/en/operations/system-tables.md:27]: here too

File: docs/en/operations/system-tables.md [hunk 7]
```diff
@@ -5,11 +5,40 @@ toc_title: System Tables
...
+
+**procfs**
+
+If ClickHouse binary file doesn't `CAP_NET_ADMIN` capability, the server tries to fallback to `ProcfsMetricsProvider`. `ProcfsMetricsProvider` allows to collect per-query system metrics (for CPU and I/O).
```
Comment (blinkov) [inline docs/en/operations/system-tables.md:31]:
```suggestion
If ClickHouse server doesn't have `CAP_NET_ADMIN` capability, it tries to fall back to `ProcfsMetricsProvider`. `ProcfsMetricsProvider` allows collecting per-query system metrics (for CPU and I/O).
```

File: docs/en/operations/system-tables.md [hunk 8]
```diff
@@ -5,11 +5,40 @@ toc_title: System Tables
...
+
+If procfs is supported and enabled on the system, ClickHouse server collects:
```
Comment (blinkov) [inline docs/en/operations/system-tables.md:33]:
```suggestion
If procfs is supported and enabled on the system, ClickHouse server collects these metrics:
```

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/en/operations/system-tables.md (hunk 2 above)
Comment (qoega) [reply docs/en/operations/system-tables.md:17]: query_log, metric_log, trace_log and probably others store data on disk. You can check data directory if you have ClickHouse running

**Pull request merged**